app/frontend/src/store/modules/form.js

```diff
@@ -309,6 +314,18 @@ export default {
         }, { root: true });
       }
     },
+    async fetchSubmissionUsers({ commit, dispatch }, { formSubmissionId }) {
```

Comment:
Yeah if it's a single one it can be a single stand alone argument, though if it's more than 1 you have to do an object like that. If you look in the file we've been a bit inconsistent for single args so this one could go either way fine.